target/linux/mediatek/image/filogic.mk

```diff
@@ -8,6 +8,15 @@ define Image/Prepare
 	echo -ne '\xde\xad\xc0\xde' > $(KDIR)/ubi_mark
 endef
 
+define Build/fit-with-netgear-top-level-rootfs-node
+	$(call Build/fit-its,$(1))
+	$(TOPDIR)/scripts/gen_netgear_rootfs_node.sh $(KERNEL_BUILD_DIR)/root.squashfs > $@.rootfs;
+	awk '/configurations/ { system("cat $@.rootfs") } 1' $@.its > $@.its.tmp;
```

Comment:
I have not checked thoroughly how the final its should look like, but I think you can forego the possibly fragile string manipulation and simply extend the existing node(s) by appending to the its.